src/components/QuranReader/TranslationView/TranslationText/index.tsx

```
@@ -155,11 +160,17 @@ const TranslationText: React.FC<Props> = ({
   return (
     <div className={styles[`translation-font-size-${translationFontScale}`]} translate="no">
-      <div
-        onClick={(event) => onTextClicked(event)}
-        className={classNames(styles.text, styles[langData.direction], styles[langData.font])}
-        dangerouslySetInnerHTML={{ __html: text }}
-      />
+      {chapterName && reference && (
+        <div
+          onClick={(event) => onTextClicked(event)}
+          className={classNames(styles.text, styles[langData.font])}
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)